### ISSUE TITLE
add hints in editor for first time users (à la crayon physics)

### ISSUE THREAD

**Issue by AtiX** [ad-si/brickify#248]
hint overlay like
" select this brush "
" now brush here --> X"
for first time users

**Comment by Steditor** [comment]
Maybe in connection with #593: Provide hook for 'first time use hints', provide plugin that shows and hides the hints and memorizes the states in a cookie/local storage/session/...

**Comment by adius** [comment]
This is fixed, isn't it?

**Comment by Steditor** [comment]
Nope. We only have rough hints which mouse buttons to use, but we don't have a crayon-physics-like tutorial.
